stripper/ze_sisyphus_v1.cfg

```diff
@@ -12,7 +12,7 @@ modify:
 	}
 	replace:
 	{
-		"vscripts" "gfl/sisyphus_magic_v1.nut"
+		"vscripts" "gfl/sisyphus_magic_v2.nut"
```

Comment:
Update the filenames in the comment as well.

Reply:
good catch